src/services/config/config.service.ts

```diff
@@ -94,7 +95,14 @@ export class ConfigService implements IKubeDaemonSecurityConfigService, IKubeCon
 
         const serviceUrl = this.buildServiceUrl(configName);
 
-        this.config = new UnixConfig(projectName, configName, configDir, isSystemTest, serviceUrl);
+        if (process.platform === 'win32') {
+            this.config = new WindowsConfig(logger, projectName, configName, configDir, isSystemTest, serviceUrl);
+        } else if (process.platform === 'linux' || process.platform === 'darwin') {
+            this.config = new UnixConfig(projectName, configName, configDir, isSystemTest, serviceUrl);
+        } else {
+            logger.error(`Unable to find config: unsupported operating system ${process.platform}`);
+        }
```

Comment:
IMO there's no reason to exclude aix, freebsd, openbsd, and sunos from ZLI support; they're all unix-based, so this can just be 
```suggestion
        } else {
            this.config = new UnixConfig(projectName, configName, configDir, isSystemTest, serviceUrl);
        }
```

Reply:
I don't disagree with this but I don't know if we do want to make this change because it shifts some os's into an "implicit support" category which we don't have any explicit testing for in our matrix. I think we should get a thumbs up from the team before making the change and then make sure it's consistent across all of these such if statements.

Reply:
For what is worth, I agree that we should only be using what we are testing. There should be a clear, Product-specified list of OSs we support and have our code support them and test for them in some way. It may be cumbersome to test for them in every PR so we could just test all of them in the CI, but still, that's the proper way of doing things

Reply:
alright, conclusion from the meeting sides with @johncmerfeld. We can easily put all of our unix support into an "else" statement.

Reply:
2077e00